internal/controllers/core/tiltfile/reconciler.go

```diff
@@ -89,3 +267,77 @@ func indexTiltfile(obj client.Object) []indexer.Key {
 	}
 	return result
 }
+
+func requiresDocker(tlr tiltfile.TiltfileLoadResult) bool {
+	if tlr.Orchestrator() == model.OrchestratorDC {
+		return true
+	}
+
+	for _, m := range tlr.Manifests {
+		for _, iTarget := range m.ImageTargets {
+			if iTarget.IsDockerBuild() {
+				return true
+			}
+		}
+	}
+
+	return false
+}
+
+// Represent the steps of Tiltfile execution.
+type runStep int
+
+const (
+	// Nothing needs to be done.
+	runStepNone runStep = iota
```

Comment:
Is this more accurately `// Tiltfile execution is waiting to start`?

Reply:
done